modules/olmv1-adding-a-catalog.adoc

```diff
@@ -63,7 +74,7 @@ redhat-operators      20s
 +
 [source,terminal]
 ----
-$ oc get catalogs.catalogd.operatorframework.io -o yaml
+$ oc describe catalog
```

Comment:
The example output is still the output of command 'oc get catalogs.catalogd.operatorframework.io -o yaml'.
The example output should be updated or change back to 'oc get catalogs.catalogd.operatorframework.io -o yaml'

Reply:
Thank you, @Xia-Zhao-rh. Feedback addressed.